protobuf/flow/execution/execution.proto

```diff
@@ -74,7 +76,15 @@ message GetEventsForBlockIDsRequest {
     repeated bytes block_ids = 2;
 }
 
-message GetEventsForBlockIDTransactionIDRequest {
+// Transactions
+
+message GetTransactionResultRequest {
     bytes block_id = 1;
     bytes transaction_id = 2;
 }
+
+message GetTransactionResultResponse {
+    uint32 status_code = 2;
```

Comment:
Including **status_code** and **error** in the response is the main reason for the change.